roadmap.md

```diff
@@ -1,60 +1,41 @@
+- *Long Term*: The team is investigating for the future but don’t have a clear solution in mind. This is the area most in flux as investigations can change feasibility & priority. Do not make plans reliant on these features as we cannot guarantee they will be implemented. 
 
-## Current Quarter
+## Timelines
 
-### CY2022Q1 / FY2022Q3
+### Near Term
+- Node token caching support with Redis 
+- Internal pipeline improvements for easier release cadence 
+- Deprecation of passport-azure-ad and replacement with a new lightweight token validator 
+- Updates required by the recent United State’s Executive Order on cybersecurity: Executive Order on Improving the Nation's Cybersecurity | The White House 
 
-| Library         | Deliverable                       | Status      | ETA       | Notes                                                        |
-| :-------------- | :-------------------------------- | :---------- | :-------- | :----------------------------------------------------------- |
-| MSAL Browser    | Telemetry interface               | In Progress | Mar 2022  | To provide support for customers that are interested in library performance |
-| MSAL Browser    | Security improvements             | In Progress | July 2022 | Improvements for Chrome and Edge on Windows to leverage existing device accounts |
-| MSAL Browser    | Resiliency improvements           | In Progress | Mar 2022  | Improvements around accessing the metadata endpoint to improve library reliability |
-| MSAL Node       | Distributed cache support         | In Progress | Mar 2022  | Improvements and samples to better support distributed caches like Redis in MSAL Node |
- 
+### Mid Term
+- Improvements on Node documentation & samples  
+- Ivy Library Distribution of MSAL-angular 
+- Angular 14 support 
+- Sustainable strategy to release adoption drivers like library dependency changes  
+- Software bound RT Pop support  
+- First Party Sets support  
 
-### CY2021Q4 / FY2022Q2
+### Long Term
+- IDP agnostic support  
+- Enhancing MSALGuard 
+- Reducing package size 
+- Increased support for the AAD Backup Auth Services  
+- Browser federated identity standards (FedCM, Storage Access API, etc) 
 
-| Library         | Deliverable                       | Status      | ETA       | Notes                                                        |
-| :-------------- | :-------------------------------- | :---------- | :-------- | :----------------------------------------------------------- |
-| MSAL Browser    | General performance improvements  | In Progress | Dec 2021  | End to end review on token performance and proposals for optimizations. |
 
-## Past Quarters
+### Past
+This is not a comprehensive list of all completed features, but rather a way to note when something moved from near term to complete recently. You can view all past Release Notes of MSAL.js (here)[].
```

Comment:
I'd also suggest we can get by with just links to the changelogs, rather than listing out released features. Less to maintain :)

Reply:
My thinking was to move anything released from "near term" to "past" at least until the next roadmap update. My hesitancy is if a feature is removed from the list, people may not know if it was removed because we released it or because we decided not to implement it. It wouldn't be a long-lived list in past. Open to being convinced a link to release notes is enough for people to go looking for a specific feature they no longer see on the list.